File: MemoMaster/app/src/main/java/memomaster/lunastratos/com/memomaster/Fragments/WriteFragment.java
package memomaster.lunastratos.com.memomaster.Fragments;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import memomaster.lunastratos.com.memomaster.R;

public class WriteFragment extends Fragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View v = inflater.inflate(R.layout.write_layout,container ,false);
        
        return v;
    }
    @Override
    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);


    }
}

File: WhiteAlbum/app/src/main/java/memokotlin/lunastratos/com/whitealbum/Fragment/WriteFragment.kt
package memokotlin.lunastratos.com.whitealbum.Fragment


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import memokotlin.lunastratos.com.whitealbum.R

class WriteFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        val v: View = inflater.inflate(R.layout.fragment_home, container, false)

        return v
    }
}